deployment/puppet/refinery/manifests/python.pp

```diff
@@ -72,12 +79,18 @@
     }
   }
 
-  package { 'virtualenvwrapper': }
+  exec { 'get virtualenvwrapper':
+    command     => "python3 -m pip install virtualenvwrapper",
+    path        => [ '/bin/', '/sbin/' , '/usr/bin/', '/usr/sbin/' ],
+    user        => $app_user,
+    group       => $app_group,
+    require     => Python::Virtualenv[$virtualenv],
+  }
   ->
   file_line { "virtualenvwrapper_config":
-    path    => "/home/${app_user}/.profile",
-    line    => "source /etc/bash_completion.d/virtualenvwrapper",
-    require => Python::Virtualenv[$virtualenv],
+    path        => "/home/${app_user}/.profile",
+    line        => "export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3 && export WORKON_HOME=/home/${app_user}/.virtualenvs/ && source ./.local/bin/virtualenvwrapper.sh",
```

Comment:
I don't think this line should be changed at all if virtualenvwrapper is installed by APT.